corehq/apps/app_manager/tests/views/test_apply_patch.py

```diff
@@ -0,0 +1,103 @@
+import difflib
+
+from ...views.forms import apply_patch
+
+
+def test_empty_patch():
+    diff(apply_patch("", ""), "")
+
+
+def test_vellum_xml_patch():
+    # Form and patch generated by creating a simple form in stand-alone Vellum
+    # and then inspecting a save call in src/core.js with the debugger.
+    diff(apply_patch(PATCH, FORM_XML), PATCHED_FORM)
+
+
+def diff(actual, expected):
+    if actual != expected:
+        diff = difflib.unified_diff(
+            expected.splitlines(1),
```

Comment:
[The docs](https://www.programiz.com/python-programming/methods/string/splitlines) suggest splitlines is just taking a boolean parameter? Is there a reason to use `1` rather than `splitlines(keepends=True)` or `splitlines(True)`?

Reply:
I copied this from another test utility.

https://github.com/dimagi/commcare-hq/blob/c3b16ae22854572fe64142418b698c9a17502d33/corehq/apps/app_manager/tests/util.py#L124-L129

I'll fix it in a follow-up PR.